mbof/views.py

```diff
@@ -27,60 +28,107 @@ class CurrentUserViewSet(UserViewSet):
 
 class EventViewSet(viewsets.ModelViewSet):
     """
-    API endpoint that allows users to be viewed or edited.
+    API endpoint that allows Events to be viewed or edited.
     """
     queryset = Event.objects.all().order_by('postingTime')
     serializer_class = EventSerializer
 
+    def perform_create(self, serializer):
+        me = self.request.user.__str__()
```

Comment:
This method repeats a lot of what `log_viewer` does in the other classes. And those methods are both pretty repetitive.

You may just want one module-level method that accepts the request, and logs for you...

```
def log_action(request):
    me = request.user.__str__()
    # ...
```